test/e2e/playwright.config.ts

```diff
@@ -0,0 +1,70 @@
+import { defineConfig, devices } from 'playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// import dotenv from 'dotenv';
+// import path from 'path';
+// dotenv.config({ path: path.resolve(__dirname, '.env') });
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig( {
+	testDir: './specs',
+	/* Run tests in files in parallel */
+	fullyParallel: true,
+	/* Fail the build on CI if you accidentally left test.only in the source code. */
+	forbidOnly: !! process.env.CI,
+	/* Retry on CI only */
+	retries: process.env.CI ? 1 : 0,
+	/* Workers should use what is available */
+	workers: '100%',
+	/* Reporter to use. See https://playwright.dev/docs/test-reporters */
+	reporter: process.env.CI ? 'github' : 'html',
+	/* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+	use: {
+		/* Base URL to use in actions like `await page.goto('/')`. */
+		// baseURL: 'http://localhost:3000',
+
+		/* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
+		trace: 'on-first-retry',
+	},
+
+	/* Configure projects for major browsers */
+	// See https://github.com/microsoft/playwright/blob/main/packages/playwright-core/src/server/deviceDescriptorsSource.json */
+	projects: [
```

Comment:
How much of these do we need? For example, do we need both "Pixel 7" and "Galaxy S24" as representative samples for Android coverage? How much do each of these entries slow down the build or occupy CI runner time?

Reply:
Yes I tend to agree this is too many, I guess I was trying to demonstrate what was possible, and CI doesn't need to run them all, we can use --project arguments to limit them. 

The existing tests just use two, but the Pixel 4a is about 5 years old now.

```
case 'mobile':
	deviceName = 'Pixel 4a (5G)';
	break;
case 'desktop':
	deviceName = 'Desktop Chrome HiDPI';
	break;
```

I can update the main playwright test tasks to only run a desktop and mobile one, and add an option to use all locally.